db/schema.rb

```diff
@@ -10,21 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_04_12_005617) do
+ActiveRecord::Schema.define(version: 2019_04_28_013301) do
 
   create_table "questions", force: :cascade do |t|
     t.text "options"
     t.string "title"
     t.text "next_question"
-    t.boolean "can_skip"
```

Comment:
can u not skip anymore

Reply:
no, with the current implementation there won't be too many questions to begin with so we just omitted it for now to avoid confusion